traffic_ops/testing/api_contract/v4/test_cdns.py

```diff
@@ -0,0 +1,68 @@
+"""Api Contract Test Case for cdns endpoint"""
+import json
+import logging
+import pytest
+
+# Create and configure logger
+logger = logging.getLogger()
+
+
+@pytest.fixture(name="get_cdn_data")
+def get_cdn_prereq_data():
+    """PyTest Fixture to store prereq data for cdns endpoint"""
+    # Response keys for cdns endpoint
+    with open('prerequisite_data.json', encoding="utf-8", mode='r') as prereq_file:
+        data = json.load(prereq_file)
+    cdn_data = data["cdns"]
+    return cdn_data
+
+
+def test_get_cdn(to_session, get_cdn_data, cdn_prereq):
```

Comment:
missing argument types, function return types

Reply:
Test function doesn't have return value. Please correct me if i'm wrong. And argument types were added before.

Reply:
Every function returns something, but if your function has no explicit `return` statement the returned value is simply `None`, which would be the return type of this function.

> And argument types were added before.

I see types mentioned in the docstrings, but more importantly they're missing on the actual arguments

Reply:
Changed it in actual arguments.